tests/test_export_torchscript.py

```diff
@@ -28,6 +28,7 @@
     convert_scripted_instances,
     get_sample_coco_image,
     random_boxes,
+    SLOW_PUBLIC_CPU_TEST
```

Comment:
why?

Reply:
Before `SLOW_PUBLIC_CPU_TEST` was defined here, but I moved to the new place to be used by `test_export_onnx.py`. I didnt use yet as I didnt measure the export time on CI just yet